wrappers/dotnet/indy-sdk-dotnet-test/AgentTests/AgentCloseConnectionTest.cs

```diff
@@ -12,12 +13,12 @@ public class AgentCloseConnectionTest : AgentIntegrationTestBase
         {
             var endpoint = "127.0.0.1:9603";
 
-            var myDid = await Signus.CreateAndStoreMyDidAsync(_wallet, "{}");
+            var myDid = await SignUs.CreateAndStoreMyDidAsync(_wallet, "{}");
```

Comment:
Initially `us` is just a suffix for `sign`, let's change back `SignUs` to `Signus`.

Reply:
OK.  The only reason I changed it is that it's unclear what the pronunciation of "Signus" is; it could be with a hard 'g'.  Slava mentioned that the correct pronunciation is "sign us" so I change the case to reflect that.  If after reviewing this the preference is still to roll back to the original name I don't see a problem.

Reply:
> If after reviewing this the preference is still to roll back to the original name I don't see a problem.

yes, please roll back